### PR TITLE
numa_hmat: Add acpi hmat test

### PR DESCRIPTION
**Pull request by vivianQizhu**
id: 1859029
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
Depends on: https://github.com/avocado-framework/avocado-vt/pull/2849

**Comment by vivianQizhu** [comment]
@huangyum Please help review, thanks.
 (1/1) type_specific.io-github-autotest-qemu.numa_hmat: PASS (42.86 s)

File: qemu/tests/cfg/numa_hmat.cfg [hunk 1]
```diff
@@ -0,0 +1,80 @@
+- numa_hmat:
+    type = numa_hmat
+    only x86_64
+    start_vm = no
+    machine_type_extra_params = 'hmat=on'
```
Comment (huangyum) [inline qemu/tests/cfg/numa_hmat.cfg:5]: Required qemu>=4.2.

**Comment by vivianQizhu** [comment]
@huangyum Updated, thanks.

File: qemu/tests/numa_hmat.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        params['numa_hmat_caches_size_hmat_cache2'] = '40K'
+        params['numa_hmat_caches_size_hmat_cache3'] = '80K'
+        params['numa_hmat_caches_size_hmat_cache4'] = '70K'
+        params['numa_hmat_caches_size_hmat_cache5'] = '60K'
+    else:
+        params['numa_hmat_caches_size_hmat_cache1'] = '40K'
+        params['numa_hmat_caches_size_hmat_cache2'] = '50K'
+        params['numa_hmat_caches_size_hmat_cache3'] = '60K'
+        params['numa_hmat_caches_size_hmat_cache4'] = '70K'
+        params['numa_hmat_caches_size_hmat_cache5'] = '80K'
```
Comment (huangyum) [inline qemu/tests/numa_hmat.py:32]: Test failed.

22:50:19 INFO | [qemu output] qemu-kvm: -numa hmat-cache,node-id=1,level=2,size=40K,associativity=direct,policy=write-back,line=8: Invalid size=40960, the size of level=2 should be larger than the size(51200) of level=1
22:50:19 INFO | [qemu output] (Process terminated with status 1)

**Comment by vivianQizhu** [comment]
@huangyum Updated, please help check, thanks.

**Review by huangyum — APPROVED** [review]
ACK 
(1/1) type_specific.io-github-autotest-qemu.numa_hmat: PASS (44.06 s)

**Pull request merged**